src/main/scala/beam/replanning/AddSupplementaryTrips.scala

```diff
@@ -16,28 +16,29 @@ class AddSupplementaryTrips @Inject()(beamConfig: BeamConfig) extends PlansStrat
   private val log = LoggerFactory.getLogger(classOf[AddSupplementaryTrips])
 
   override def run(person: HasPlansAndId[Plan, Person]): Unit = {
-    if (beamConfig.beam.agentsim.agents.tripBehaviors.mulitnomialLogit.generate_secondary_activities) {
-      log.debug("Before Replanning AddNewActivities: Person-" + person.getId + " - " + person.getPlans.size())
-      ReplanningUtil.makeExperiencedMobSimCompatible(person)
+    if (!beamConfig.beam.agentsim.agents.tripBehaviors.mulitnomialLogit.generate_secondary_activities) {
+      return
```

Comment:
yes, because it was if block with only branch

Reply:
In this specific case it might be ok...but it leads to a slippery slope. I would prefer to go back to the single branch. One solution if you want something cleaner would be to break it into two methods:

```
def run...
  if(xyz) performRun
def performRun...
   //do stuff that was in run
```

That gives you less brackets over a large span